packages/calcite-components/src/components/navigation-user/navigation-user.scss

```diff
@@ -1,39 +1,39 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-navigation-user-avatar-corner-radius: Specifies the corner radius of avatar.
+ * @prop --calcite-navigation-user-avatar-color: Specifies the icon color of avatar.
+ * @prop --calcite-navigation-background-color: Specifies the component's background color.
+ * @prop --calcite-navigation-user-border-color: Specifies the components's border color.
+ * @prop --calcite-navigation-user-full-name-text-color: Specifies the component's full name text color.
+ * @prop --calcite-navigation-user-name-text-color: Specifies the component's username text color.
+ *
+ */
+
 :host {
   @apply inline-flex outline-none;
   & .button {
-    background-color: transparent;
-    border: none;
-    @apply font-sans
-      flex
+    @apply flex
       m-0
       items-center
       justify-center
       cursor-pointer
       transition-default
       focus-base
+      font-sans
       text-0h;
-    border-block-end: 2px solid transparent;
+    border: none;
+    background-color: var(
+      --calcite-navigation-background-color,
+      var(--calcite-internal-navigation-user-background-color, transparent)
+    );
+    border-block-end: 2px solid
```

Comment:
We've identified a design pattern that will likely be moved to semantic tokens in the future. The selection-color pattern goes
default = transparent
hover = border-1 (not always present)
press = brand

`border-block-end: 2px solid
      var(--calcite-navigation-selection-color, var(--calcite-internal-navigation-selection-color, transparent));`

Reply:
Is it suggestible to follow the `-selection-` naming here for `border-block-end` or wait for semantic tokens. 

This applies for `navigation-logo` as well.

Reply:
can you use --calcite-navigation-selection-color for both?

Reply:
`--calcite-navigation-selection-border-color` might be more meaningful here.

Reply:
>We've identified a design pattern that will likely be moved to semantic tokens in the future. The selection-color pattern goes
default = transparent
hover = border-1 (not always present)
press = brand

Had some thinking over this with Skye and don't think `selection` pattern applies here completely because we shouldn't allow user to customize `border-color` in default case when `active=false`. User's are expected to customize `border-color` only when `active=true` and no other interaction or properties should be able to change it. 

```
//default case
anchor {
border-block-end : 2px solid transparent
}
```

Perhaps using `--calcite-navigation-selection-border-color` or `--calcite-navigation-active-border-color` might be more semantic. I am leaning towards `--calcite-navigation-active-border-color` because selection on `navigation-logo` and `navigation-user` doesn't add the border and users have to explicitly set the `active` prop to `true ` to get the border added.  To differentiate the behavior prefer using `-active-` naming in token.

cc @SkyeSeitz

Reply:
in other components, the solution is to only apply `--calcite-navigation-selection-border-color` on [active] and leave it transparent for other style props